piqueserver/statusserver.py

```diff
@@ -1,144 +1,153 @@
-# Copyright (c) junk/someonesomewhere 2011.
```

Comment:
pretty sure this is still legally a derived work,  this copyright notice can't be removed. I hate these long headers too, but people are torn on if just putting a LICENSE file in your folder is legally sufficient, and the GPL license recommends adding one. Here's some options:

 - put the licence preamble and copyright info at the end of the file
 - use a spdx ID, which is just one line. I'm not entirely sure if this is enough legally, but it's better than no copyright notice and also machine readable. SPDX recommands addng the copyright header even when you're using SPDX.